test/e2e/e2e_suite_test.go

```diff
@@ -5,8 +5,8 @@ package e2e
 
 import (
 	"context"
+	errors "errors"
```

Comment:
(optional nit) I'm surprised this one didn't get cleaned up...

```suggestion
	"errors"
```